src/dialect/awsAthenaDialect.ts

```diff
@@ -0,0 +1,167 @@
+import { Duration, Timezone } from 'chronoshift';
+import { PlyType, PlyTypeSimple } from '../types';
+import { SQLDialect } from './baseDialect';
+
+export class AwsAthenaDialect extends SQLDialect {
+  static TIME_BUCKETING: Record<string, string> = {
+    PT1S: '%Y-%m-%d %H:%i:%SZ',
+    PT1M: '%Y-%m-%d %H:%i:00Z',
+    PT1H: '%Y-%m-%d %H:00:00Z',
+    P1D: '%Y-%m-%d 00:00:00Z',
+    P1M: '%Y-%m-01 00:00:00Z',
+    P1Y: '%Y-01-01 00:00:00Z'
+  };
```

Comment:
I have a question. Will let's say P2Y work as well ? Or it's only for particular time ?

Reply:
P2Y will throw `unsupported duration` error for now